File: extract_was.py
import re

with open('c:/workspace/itstory/was.html', 'r', encoding='utf-8') as f:
    html = f.read()

matches = re.findall(r'<div class="event-detail__(?:story|notes)">.*?</div>', html, re.DOTALL)
for i, m in enumerate(matches):
    print(f'--- Match {i} ---')
    print(m)

File: scripts/extract_was.js
const fs = require('fs');
const html = fs.readFileSync('c:/workspace/itstory/was.html', 'utf8');
const matches = html.match(/<div class="event-detail__(?:story|notes)">[\s\S]*?<\/div>/g);
if (matches) {
    matches.forEach((m, i) => {
        console.log(`--- Match ${i} ---`);
        console.log(m);
    });
} else {
    console.log('No matches found.');
}
